plugins/locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/mylocationlayer/MyLocationLayerConstants.java

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.mapboxsdk.plugins.mylocationlayer;
+
+/**
+ * Contains all the constants being used for the My Location layer.
+ *
+ * @since 0.1.0
+ */
+class MyLocationLayerConstants {
```

Comment:
Are these constants used all around the codebase?
If not, these should be encapsulated by the respective classes themselves.